docs/LibraryEvolution.rst

```diff
@@ -781,9 +756,9 @@ Initializers
 
 For the most part enum initializers are treated exactly like top-level
 functions. They permit all of the same modifications and can also be marked
-``@inlineable``, with the same restrictions. Unlike struct initializers, enum
+``@inlinable``, with the same restrictions. Unlike struct initializers, enum
 initializers do not always need to delegate to another initializer, even if
-they are inlineable or declared in a separate module.
+they are inlinable or declared in a separate module.
```

Comment:
Enum initializers either delegate to another initializer or assign to `self`. Both are effectively equivalent.

Reply:
Yeah. I didn't feel like that was interesting to call out here.